.github/workflows/gh-linux_mac.yml

```diff
@@ -0,0 +1,360 @@
+name: builds
+on:
+  push:
+#    branches:
+#      - dev
+#      - release-*
+#      - refs/tags/*
+  pull_request:
+    branches:
+      - '*'  # must quote since "*" is a YAML reserved character; we want a string
+env:
+  BACKWARDS_COMPATIBILITY_ARRAYS: OFF
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    if: ${{ startsWith(github.ref , 'refs/tags') != true && startsWith(github.ref , 'build-') != true }}
+    timeout-minutes: 90
+    strategy:
+      matrix:
+          os: [ubuntu-16.04, macOS-10.14]
+          config: [S3, GCS, AZURE, HDFS, ASAN, SERIALIZATION]
+          exclude:
+            - {os: macOS-10.14, config: S3}
```

Comment:
Remove -- currently, AzurePipelines just calls our macos-s3 test "MacOS". We still want to test this.